packages/apps/composer-app/src/components/Placeholder.tsx

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React, { useEffect, useState } from 'react';
+
+import { DXOS } from '@dxos/brand';
+import { Status, ThemeProvider } from '@dxos/react-ui';
+import { defaultTx, getSize, mx } from '@dxos/react-ui-theme';
+
+export const Placeholder = () => {
+  const [className, setClassName] = useState<string>();
+  useEffect(() => {
+    setTimeout(() => {
+      setClassName('transition duration-1000 rotate-180 scale-100 opacity-25 blur-none');
+    }, 100);
+  }, []);
+
+  return (
+    <ThemeProvider tx={defaultTx}>
+      <div className='flex flex-col bs-[100dvh] justify-center items-center'>
+        <Status indeterminate aria-label='Initializing' classNames='block is-full' />
+        <div className={mx('flex grow items-center scale-50 opacity-0 blur-xl', className)}>
```

Comment:
Generally `div`s should not need to be `scale`’d, did something motivate this over setting a plain size?

Reply:
it sets the initial size prior to the transition being applied in the timeout above. is there a better way to do this?

Reply:
Ah I think I didn't realize this was part of the animation, it's fine if so.